it/controllers/updateRegisteredDetails/WarehouseDetailsControllerISpec.scala

```diff
@@ -0,0 +1,285 @@
+package controllers.updateRegisteredDetails
```

Comment:
can we have assertions in here of what the address looks when page is rendered on 200
i.e.
when user has 1 (hopefully more than 1) warehouse, here is the address that is displayed
when user has No warehouse what is displayed